### PR TITLE
Add search bar to every page except the front page

### PR DESCRIPTION
**Pull request by carlinmack**
Closing #25 

@Daniel-Mietchen 😁

### CODE REVIEW

**Comment by carlinmack** [comment]
I've made the word Scholia bold in the header to distinguish it a bit, is there a logo? #878

**Comment by fnielsen** [comment]
There are some logos here: https://commons.wikimedia.org/wiki/Category:Scholia

**Comment by carlinmack** [comment]
Added logo to website, will upload the wordmark and other variations to commons

**Comment by carlinmack** [comment]
## Preview

Homepage

image

Searching

image

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -201,6 +201,24 @@
...
+          action: "wbsearchentities",
+          language: "en",
+          uselang: "en",
+          format: "json",
+          strictlanguage: true,
+        },
+        function(data) {
+          $("#searchresult").empty();
+          $.each(data.search, function(i, item) {
+            $("#searchresult").append("<div><a href='" + item.title + "'>" + item.label + "</a> - " + item.description + "</div>");
```
Comment (fnielsen) [inline scholia/app/templates/base.html]: I think with need a "./" here.

Comment (fnielsen) [reply scholia/app/templates/base.html]: No that is not enough. There is a wrong redirect

Comment (fnielsen) [reply scholia/app/templates/base.html]: I think there might also be an escape issue

Comment (carlinmack) [reply scholia/app/templates/base.html]: Where in this line? it seems to work fine on my end

Comment (fnielsen) [reply scholia/app/templates/base.html]: I have just added the escape myself.

**Comment by carlinmack** [comment]
I didn't realise this was used, can you

**Pull request closed without merging**